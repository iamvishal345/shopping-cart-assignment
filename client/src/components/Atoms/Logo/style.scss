@import "assets/scss/_breakpoints.scss";
// .logo-container {
//   font-size: 24px;
//   text-align: center;
//   font-weight: 700;
//   width: fit-content;
//   position: relative;
//   width: 130px;
//   .box {
//     position: absolute;
//   }
//   .one {
//     width: 55px;
//     height: 55px;
//     top: 50%;
//     background-color: #c12856;
//     transform: translateY(-50%);
//   }
//   .two {
//     width: 60px;
//     height: 60px;
//     background-color: #e3913f;
//     left: 46%;
//     top: 32%;
//     transform: translate(-50%, -15px);
//     mix-blend-mode: overlay;
//   }
//   .three {
//     width: 60px;
//     height: 60px;
//     right: 0;
//     background-color: #79d7e1;
//     top: 50%;
//     transform: translateY(-50%);
//     mix-blend-mode: darken;
//   }
//   .brand {
//     top: 6%;
//     position: relative;
//     color: #ffffff;
//   }
// }

.logo-image {
  @include xm {
    height: 40px;
  }
}
