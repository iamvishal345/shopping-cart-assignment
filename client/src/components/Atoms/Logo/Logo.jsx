import React from "react";
import "./style.scss";

export const Logo = () => (
  // <div className="logo-container">
  //   <div className="box one"></div>
  //   <div className="box two"></div>
  //   <div className="box three"></div>
  //   <span className="brand">
  //     SABKA <br /> BAZAAR
  //   </span>
  // </div>
  <picture>
    <source srcSet="/static/images/logo.png" media="(max-width: 767.98px)" />
    <img
      className="logo-image"
      height="50"
      src="/static/images/logo_2x.png"
      alt="Sabka Bazaar"
    />
  </picture>
);
