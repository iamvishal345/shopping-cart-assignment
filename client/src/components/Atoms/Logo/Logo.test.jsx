import React from "react";
import { shallow } from "enzyme";
import { Logo } from "./Logo";

let wrapped = shallow(<Logo />);

describe("Logo", () => {
  it("should match the snapshot", () => {
    expect(wrapped).toMatchSnapshot();
  });
});
