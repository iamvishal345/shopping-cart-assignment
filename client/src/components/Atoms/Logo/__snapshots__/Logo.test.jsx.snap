// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should match the snapshot 1`] = `
<picture>
  <source
    media="(max-width: 767.98px)"
    srcSet="/static/images/logo.png"
  />
  <img
    alt="Sabka Bazaar"
    className="logo-image"
    height="50"
    src="/static/images/logo_2x.png"
  />
</picture>
`;
